### DISCUSSION TITLE
[Feature Req:] Min/Max/Average speeds

### DISCUSSION THREAD

**Discussion by 3rdStng** [alexjustesen/speedtest-tracker#650] [Ideas]
Recently learned of this speedtest tracker docker from the original github that this build forked from. Having continuous issues with the other branch, I switched to this one and had zero issues or extra steps to get things running. Love the newer UI look too. 

Feature request, that I enjoyed from the other branch was a Min/Max/Average speeds being displayed in the UI. Would that be possible to do?

**Comment by alexjustesen** [comment]
I really appreciate the positive feedback! I don't see why not, I'm digging through a mountain of client work at the moment but it's a good improvement to the analytics

**Reply by ZoltrixGFC** [reply]
Yes showing the top speeds/pings recorded (of all time) would be great!

**Comment by jakehall12** [comment]
Having just started using this docker, and loving it currently, I think one thing missing from the dashboard is an Average speed over the about of time selected. Would be great to see a feature for this added at some stage.

**Reply by alexjustesen** [reply]
Thanks for the suggestion, a future release has plans for chart improvements as I'm currently focused on Docker image performance and stability.

**Reply by MeuricW** [reply]
Has there been any traction on adding averages since August?

**Reply by alexjustesen** [reply]
Any progress on changes to charting and visualizations can be tracked here: https://github.com/alexjustesen/speedtest-tracker/milestone/5
